include-dependencies.js

```diff
@@ -52,11 +54,33 @@ module.exports = class IncludeDependencies {
     service.package = service.package || {};
     service.package.exclude = union(service.package.exclude, ['node_modules/**']);
 
+    this.processIncludes(functionObject);
+
     if (runtime.match(/nodejs*/)) {
       this.processNodeFunction(functionObject);
     }
   }
 
+  processIncludes(functionObject) {
+    const service = this.serverless.service;
+    if (!service.package || !service.package.include || !service.package.include.length) {
+      return;
+    }
+    service.package.include.forEach(includeGlob => {
+      glob.sync(path.join(this.serverless.config.servicePath, includeGlob)).forEach(filePath => {
+        var stat = fs.statSync(filePath);
+        if (stat && stat.isFile()) {
+          const dependencies = this.getDependencies(filePath, service.package.exclude);
+          if (service.package.individually) {
```

Comment:
I think functions can have include on their own too? Do you mind supporting this?

Reply:
👍 will do

I'm away til Monday but will look at this when I get back.